proposals/mixins.py

```diff
@@ -106,7 +107,17 @@ def get_pdf_response(self, context, dest=None, **response_kwargs):
             dest = response
 
         # find the template and render it.
-        template = get_template(self.template_name)
+        template_names = self.get_template_names()
+
+        if not (type(template_names) is tuple or list):
```

Comment:
I don't think this works as intended?
The 'or' here is not part of the 'is' statement, but rather a 'what if the previous thing is false'. 
I did a small sanity check, and it seems to agree with me:
```
>>> x = []
>>> type(x) is tuple or list
<class 'list'>
>>> x = ()
>>> type(x) is tuple or list
True
```

I'd use this instead: 
```python
if type(template_names) not in [tuple, list]:
```
Alternatively:
```python
from collections.abc import Iterable
[..]
if not isinstance(template_names, Iterable):
```

Reply:
Good catch. The logic made no sense. It now checks for an instance of `Iterable`.